### PR TITLE
Add ask_for_logout and call function in apply_current

### PR DESCRIPTION
**Pull request by JeffLabonte**
The question will be asked only when the configuration will set to a value

This is related to issue #22

### CODE REVIEW

File: prime-select.sh [hunk 1]
```diff
@@ -219,12 +232,12 @@ function apply_current {
             # this can happen if user set nvidia but changed to "Integrated only" in BIOS (possible on some MUXED Optimus laptops)
             # in that case the NVIDIA card is not visible to the system and we must switch to intel
             
-            logging "Forcing intel due to NVIDIA card not found"
-            current_type="intel"
+            logging "Forcing intel due to NVIDIA card not found" current_type="intel"
```
Comment (sndirsch) [inline prime-select.sh]: logging call and setting current_type in the same line?

File: prime-select.sh [hunk 2]
```diff
@@ -299,8 +313,9 @@ case $type in
         fi
         
         if (( service_test == 0)); then
-	    
-	    if ! [ -f /etc/systemd/system/multi-user.target.wants/prime-select.service ]; then
+
+        if ! [ -f /etc/systemd/system/multi-user.target.wants/prime-select.service ]; then
```
Comment (sndirsch) [inline prime-select.sh]: Why this change?

File: prime-select.sh [hunk 3]
```diff
@@ -219,12 +232,12 @@ function apply_current {
...
             # in that case the NVIDIA card is not visible to the system and we must switch to intel
             
-            logging "Forcing intel due to NVIDIA card not found"
-            current_type="intel"
+            logging "Forcing intel due to NVIDIA card not found" current_type="intel"
         fi
         
         
         set_$current_type
+        ask_to_logout
```
Comment (sndirsch) [inline prime-select.sh]: Doesn't this change a previously non-interactive script to an interactive script?

File: prime-select.sh (hunk 2 above)
Comment (JeffLabonte) [reply prime-select.sh]: This hunk shouldn't be there.

Comment (JeffLabonte) [reply prime-select.sh]: This hunk shouldn't be there.

File: prime-select.sh (hunk 1 above)
Comment (JeffLabonte) [reply prime-select.sh]: This hunk shouldn't be there

File: prime-select.sh (hunk 3 above)
Comment (JeffLabonte) [reply prime-select.sh]: Yes, after asking prime-select to switch for a different from driver, it will ask you if you wish to logout. 

By default, if you press enter, it won't log you out.

File: prime-select.sh (hunk 2 above)
Comment (sndirsch) [reply prime-select.sh]: I don't like such a change in the script.

Comment (JeffLabonte) [reply prime-select.sh]: It wasn't suppose to be. I thought this hunk was removed from my commit yesterday evening. I will rebase and put it back the way it was. It wasn't intentional!

File: prime-select.sh (hunk 3 above)
Comment (sndirsch) [reply prime-select.sh]: Since users may used to use the script in a non-interactive way, I don't want to see such a change honestly. If, then only with an additional option "[--interactive]" or whatever.

**Comment by sndirsch** [comment]
Not acceptable in this form. Sorry.
